src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,56 @@
-import React from 'react';
-
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
```

Comment:
I see your Props and Movie types almost same, so you can use Omit to avoid repeat yourself:
` type IProps = Omit<Movie, 'imdbId'>`